setup.cfg

```diff
@@ -215,3 +215,8 @@ ignore_missing_imports = True
 
 [mypy-packages/fetchai/protocols/yoti/yoti_pb2]
 ignore_errors = True
+
+[darglint]
+docstring_style=sphinx
+strictness=short
```

Comment:
This means we permit single line docstrings which we do want for properties. When the docstring is more than a single line the full checks are applied.